libs/atqr/domain/src/lib/EmailService/templates/congrats.ts

```diff
@@ -2,14 +2,14 @@ import { Player } from '../../player/player.entity';
 import { Email } from '../email.service';
 import * as pug from 'pug';
 import path = require('path');
+import { pathToTemplates } from './challengeStarted';
 
-// const compileTemplate = pug.compileFile(
-//   path.join(__dirname, 'congrats.pug')
-// );
-
+const compileTemplate = pug.compileFile(
+  path.join(pathToTemplates, 'congrats.pug')
+);
 
 export class Congrats extends Email {
-  constructor(to: Player) {
+  constructor(to: Player['_email']) {
     super(to,
```

Comment:
Congrats email doesn't follow convention from other emails, template compiler must be passed to super in the constructor